quandl/get_pit_data.py

```diff
@@ -0,0 +1,70 @@
+from quandl.model.pit import Pit
+from quandl.errors.quandl_error import LimitExceededError
+from .api_config import ApiConfig
+from .message import Message
+import warnings
+import copy
+
+
+def get_pit_data(datatable_code, **options):
```

Comment:
How is point in time different from get_table as it appears a bunch of this code is the same as `get_table`. Could you show some of the usages of how `get_pit_data` will be called>

Reply:
So, that is tricky. 

They are basically the same, the difference is on the params and URL. The content is going to be the same but filtered based on the params. 

The URL is different because of this ticket https://quandl.atlassian.net/browse/DRW-203.
This is why they really look identical.